src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1796,7 +1796,7 @@ elementFormDefault="qualified">
     </xs:element>
     <xs:element name="VersionPrefix" type="msb:StringPropertyType" substitutionGroup="msb:Property" >
       <xs:annotation>
-        <xs:documentation><!-- _locID_text="VersionPrefix" _locComment="" -->Prefix that can indicate additional information at the end of the verion (i.e. beta)</xs:documentation>
+        <xs:documentation><!-- _locID_text="VersionPrefix" _locComment="" -->Prefix that can indicate additional information at the start of the version (i.e. beta)</xs:documentation>
```

Comment:
How about:

- VersionPrefix: When `Version` is not specified, `VersionPrefix` represents the first fragment of the version number (e.g. 1.0.0). The format is <VersionPrefix>[-<VersionSuffix>]
- VersionSuffix: Optional. When `Version` is not specified, `VersionSuffix` represents the second fragment of the version number (e.g. beta). The format is <VersionPrefix>[-<VersionSuffix>]